modules/bitgo/src/v2/coins/near.ts

```diff
@@ -155,10 +158,26 @@ export class Near extends BaseCoin {
    * Explain/parse transaction
    * @param params
    */
-  explainTransaction(
+  async explainTransaction(
     params: ExplainTransactionOptions,
-  ): Promise<any> {
-    throw new MethodNotImplementedError('Near recovery not implemented');
+  ): Promise<NearTransactionExplanation> {
+    const factory = accountLib.register(this.getChain(), accountLib.Near.TransactionBuilderFactory);
+    let rebuiltTransaction;
+
+    try {
+      let rawTxBase64 = params.txPrebuild.txHex;
+      if (HEX_REGEX.test(params.txPrebuild.txHex)) {
+        rawTxBase64 = Buffer.from(params.txPrebuild.txHex, 'hex').toString('base64');
+      }
```

Comment:
Maybe be can do something like
const txHex = params.txPrebuild.txHex;
const rawTxBase64 = HEX_REGEX.test(txHex) ? Buffer.from(txHex, 'hex').toString('base64') : txHex; 

I propose this because because Im kind of confused, params.txPrebuild.txHex is always a hex? or it could come as base64? if we always expect a hex, lets just the try catch do his job, delete the if.

Reply:
Well, I'm going to try this refactoring